tools/docs/presets.ts

```diff
@@ -1,23 +1,54 @@
 import { groups as presetGroups } from '../../lib/config/presets/internal';
 import { logger } from '../../lib/logger';
+import { regEx } from '../../lib/util/regex';
 import { updateFile } from '../utils';
 
 function jsUcfirst(string: string): string {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
+// matches preset names like: regexManagers and mergeConfidence
+const longPresetNameRegex = regEx(/(?<first>[a-z]+)(?<second>[A-Z][a-z]+)/);
+
+function getEditUrl(name: string): string {
+  const url =
+    'https://github.com/renovatebot/renovate/edit/main/lib/config/presets/internal/';
+
+  const groups = name.match(longPresetNameRegex);
+  if (!groups) {
+    return `${url}${name}.ts`;
+  }
+
+  return `${url}${groups[1]}-${groups[2].toLowerCase()}.ts`;
+  // switch (newName) {
+  //   case 'regexmanagers':
+  //     return `${url}regex-managers.ts`;
+  //   case 'mergeconfidence':
+  //     return `${url}merge-confidence.ts`;
+  //   case 'full config':
+  //     return `${url}config.ts`;
+  //   default:
+  //     return `${url}${newName}.ts`;
+  // }
```

Comment:
I wasn't sure which approach to use to handle the preset names like `regexManagers` and `mergeConfidence`.

They need to be formatted from `regexManagers` to `regex-managers`. 
We can use regex matching or hard-code using switch statement.

Which do you guys prefer?

Reply:
I think I prefer switch case